lib/crypto.js

```diff
@@ -633,6 +633,11 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
+
+exports.getEngines = function getEngines() {
```

Comment:
I meant if we could just do `exports.getEngines = binding.getEngines;` and avoid the wrapper function?